bin/telemetry_sim_ATS_1VisitNight

```diff
@@ -180,8 +180,11 @@ if __name__ == "__main__":
         LOGGER.info("Sending starIntegration")
         kw = {'imageSequenceName': args.imageSequenceName,
               'imageName': imageName,
-              'imageIndex': k+1,
-              'timeStamp': timeStamp,
+              'imageNumber': k+1,
```

Comment:
Sorry, I'm just not quite understanding this.
what is k? and why it is k+1 ?

Reply:
When requesting more than one exposure per sequence.